packages/reactivity/src/reactive.ts

```diff
@@ -27,13 +28,17 @@ const readonlyValues = new WeakSet<any>()
 const nonReactiveValues = new WeakSet<any>()
 
 const collectionTypes = new Set<Function>([Set, Map, WeakMap, WeakSet])
-const observableValueRE = /^\[object (?:Object|Array|Map|Set|WeakMap|WeakSet)\]$/
+const observableMap = /*#__PURE__*/ makeMap(
```

Comment:
Maybe name this as `isObservable` instead of `observableMap` since `makeMap` returns a `function` returning a `boolean`.